FrontBackMatter/SpatialCalibration.tex

```diff
@@ -299,35 +323,50 @@ \subsubsection*{The V1 model}
 projection type set to provide a balance between excitation and
 inhibition, and between afferent and lateral influences, to provide
 robust formation of activity bubbles that allows smooth maps to
-form. The function $f$ defines a variable threshold point ($\theta$)
-dependent on the average activity of the unit as described in the next
-subsection, but in all cases the gain is fixed at unity. Note that
-unlike GCAL the inhibitory projection acts divisively rather than
-subtractively.
+form. This does not mean that the model is a balanced
+excitation/inhibition model in the formal sense, indeed the default
+parameters ensure that inhibition exceeds excitation in the absence of
+strong afferent input. The function $f$ defines an adaptive threshold
+point ($\theta$) dependent on the average activity of the unit as
+described in the next subsection, but in all cases the gain is fixed
+at unity. Note that unlike GCAL the inhibitory projection acts
+divisively rather than subtractively.
 
 At the end of the 16 settling steps, the settled V1 activation pattern
-is deemed to be the V1 response to the presented pattern. At this
-point we use the V1 response to update the threshold point ($\theta$)
-of V1 neurons (using the adaptation process described below) and to
-update the afferent and lateral inhibitory weights via Hebbian
-learning. Unlike the regular GCAL model the V1 activity is not reset
-to zero instead being allowed to decay until the onset of the next
-visual input pattern.
+is deemed to be the V1 response to the presented pattern. Since this
+is a recurrent model there is no guarantee a steady state is reached
+within settling period but through manual inspection it was confirmed
+this was almost always the case. At this point we use the V1 response
+to update the threshold point ($\theta$) of V1 neurons (using the
+adaptation process described below) and to update the afferent and
+lateral inhibitory weights via Hebbian learning. Unlike the regular
+GCAL model the V1 activity is not reset to zero instead being allowed
+to decay until the onset of the next visual input pattern, which had
+no direct effect but provides a pathway to a more continuous model of
+visual responses.
 
 In order to assess the development of long-range patchy connections in
 this model we additionally add a long-range excitatory connection,
-which we let develop without any weight. This term is modeled as an
-additional multiplicative component with a constant offset of 1,
-ensuring only facilitatory modulation from the far surround. The
-activation of a model neuron is therefore given by:
+which we let develop without any actual weight. This projection
```

Comment:
which we let develop specific synaptic weights but which is given an overall strength of zero so that this projection has no effect on the actual model.